package.json

```diff
@@ -24,68 +24,66 @@
     "url": "https://github.com/educandu/educandu"
   },
   "dependencies": {
-    "@ant-design/icons": "^4.7.0",
+    "@ant-design/icons": "^4.8.0",
     "@educandu/node-jsx-loader": "^3.0.0",
-    "@sindresorhus/slugify": "^2.1.0",
+    "@sindresorhus/slugify": "^2.1.1",
     "@unly/iso3166-1": "^1.0.2",
-    "abcjs": "^6.0.3",
+    "abcjs": "^6.1.7",
     "accept-language-parser": "^1.5.0",
-    "antd": "5.0.6",
+    "antd": "5.0.7",
     "any-base": "^1.1.0",
-    "async": "^3.2.1",
-    "aws-sdk": "^2.1006.0",
-    "axios": "^0.25.0",
-    "bcrypt": "^5.0.1",
-    "classnames": "^2.3.1",
+    "async": "^3.2.4",
+    "aws-sdk": "^2.1276.0",
+    "axios": "^1.2.1",
+    "bcrypt": "^5.1.0",
+    "classnames": "^2.3.2",
     "connect-mongo": "^4.6.0",
-    "cookie-parser": "^1.4.5",
-    "cropperjs": "^1.5.12",
+    "cookie-parser": "^1.4.6",
+    "cropperjs": "^1.5.13",
     "crypto-js": "^4.1.1",
     "debounce": "^1.2.1",
     "decompress": "^4.2.1",
-    "del": "^6.0.0",
+    "del": "^7.0.0",
     "escape-string-regexp": "^5.0.0",
-    "express": "^4.17.1",
+    "express": "^4.18.2",
     "express-async-errors": "^3.1.1",
     "express-basic-auth": "^1.2.1",
-    "express-joi-validation": "^5.0.0",
-    "express-session": "^1.17.2",
+    "express-joi-validation": "^5.0.1",
+    "express-session": "^1.17.3",
     "express-useragent": "^1.0.15",
     "fast-deep-equal": "^3.1.3",
-    "flag-icon-css": "^3.5.0",
-    "glob": "^7.2.0",
+    "flag-icons": "^6.6.6",
```

Comment:
This package was renamed and development continues on the new repo now.

Reply:
:+1: